lucene/facet/src/test/org/apache/lucene/facet/taxonomy/TestTaxonomyFacetCounts.java

```diff
@@ -726,6 +743,39 @@ public void testRandom() throws Exception {
     IOUtils.close(tw, searcher.getIndexReader(), tr, indexDir, taxoDir);
   }
 
+  private static List<List<FacetLabel>> sortedFacetLabels(List<List<FacetLabel>> allfacetLabels) {
+    for (List<FacetLabel> facetLabels : allfacetLabels) {
+      Collections.sort(facetLabels);
+    }
+
+    Collections.sort(allfacetLabels, (o1, o2) -> {
+      if (o1 == null) {
```

Comment:
Hmm why are these `null` checks necessary?  Are we really seeing `null` in the argument?  Oh, I guess this legitimately happens when the hit had no facets?  Maybe add a comment?  Hmm, actually, looking at how actual and expected are populated, neither of them seems to add `null`?  One of them filters out empty list but the other does not?

Reply:
Thanks for catching this @mikemccand. I fixed the `actualLabels` to exclude empty lists. The null checks were just me being extra cautious. I realized they were unnecessary and removed them :-)